features/datachoices/src/main/java/org/opennms/features/datachoices/web/internal/ModalInjector.java

```diff
@@ -85,16 +85,16 @@ protected String getUsageStatisticsSharingModalHtml(HttpServletRequest request)
         return null;
     }
 
-    protected String getUserDataCollectionModalHtml(HttpServletRequest request) throws TemplateException, IOException {
-        String show = System.getProperty("opennms.userDataCollection.show", "true");
+    protected String getProductUpdateEnrollmentModalHtml(HttpServletRequest request) throws TemplateException, IOException {
+        String show = System.getProperty("opennms.productUpdateEnrollment.show", "true");
```

Comment:
backwards-compat with 33.0.0/33.0.1 ;)

```suggestion
        String show = System.getProperty("opennms.productUpdateEnrollment.show", System.getProperty("opennms.userDataCollection.show", "true"));
```

Reply:
Good point, I was planning on doing that :)